client/account-recovery/lost-password/lost-password-form/test/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { LostPasswordFormComponent } from '..';
+
+describe( 'LostPassword', () => {
+	it( 'should render as expected', () => {
+		const wrapper = shallow( <LostPasswordFormComponent translate={ identity } /> );
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/8955/commits/5400f382afc1479f0340cbd2a60f953d97d3ccc2